addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicBridgeHandler.java

```diff
@@ -219,51 +220,57 @@ public HomematicGateway getGateway() {
     }
 
     /**
-     * Updates the thing for the given Homematic device.
+     * Updates all things for the given Homematic device.
```

Comment:
"all things" <-> "a device". This somehow sounds as if there is some flaw in the modelling? Or is this only relevant for gatways/bridges and devices that are connected to it? Then the wording should be revised.

Reply:
With the new address parameter it's possible to have multiple things connected to one device (same address). Therefore all things must be updated.